src/routes/privacy/+page.md

```diff
@@ -0,0 +1,47 @@
+<main class="root">
+
+# Privacy policy
+
+Effective date: September 4th, 2024
+
+This Privacy Policy ("Policy") for Files ("we", "us", or "our") describes how and why we collect, store, use, and disclose information about users ("you", "your") when you use our services ("Services"), and our desktop application (the "Application", "Files"). By using our Services, you acknowledge and consent to the practices described in this Policy.
+
+## Table of Contents
```

Comment:
If we decide to add a table of contents, we could generate it from the headers and make it scroll to the relevant sections. In the meantime, let’s remove this section.